versions/n-/nanojsonc.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "41dc847f236000b8a703cf452d37ae36c39ec155",
+      "git-tree": "679fe1b8cbd17de6b07ce2ead713580e606eb839",
+      "version": "1.3.0",
+      "port-version": 0
+    },
+    {
+      "git-tree": "27de8e0e27fd55a6b832e79a6472c59fbb018829",
```

```suggestion
      "git-tree": "41dc847f236000b8a703cf452d37ae36c39ec155",
```
Don't rewrite version history

---

@vicroms Hi, thanks for looking into this. 

>> Don't rewrite version history

You mean do not execute this command `vcpkg x-add-version nanojsonc --overwrite-version`

So basically the update is just to change the SHA512 only for the vcpkg port.

P.S. Meanwhile I did some updates before your revision, so I'd request the review again. Thanks